app/views/catalog/_home_page.html.erb

```diff
@@ -62,6 +62,14 @@
             <p>Find books, media, and e-resources set aside for classes.</p>
           </div>
         </div>
+        <div class="panel panel-default">
```

Comment:
We could add a feature flag here if we don't want to expose this until we have content. I could see arguments either way.

Reply:
Sounds good. Added in a follow on commit so we can revert l8er.